Source/Scene/GroundPrimitive.js

```diff
@@ -425,7 +425,7 @@ define([
 
     function getComputeMinimumHeightFunction(primitive) {
         return function(granularity, ellipsoid) {
-            return primitive._minHeight;
+            return primitive._minHeight - 15;
```

Comment:
What view parameters?  This function takes a granularity and an ellipsoid.

There's no specific reason I chose 15.  For the code example I pasted above even subtracting 1.0 fixed the problem.  From my understanding problem happens when the shadow volume ends exactly at the height the terrain does, so maybe 1.0 would work for all cases?  I honestly don't know. @bagnell @tfili what do you think?